src/software/jetson_nano/services/motor.cpp

```diff
@@ -292,7 +358,8 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
     if (reset_detector == 2147483647)
     {
         LOG(DEBUG) << "RESET DETECTED";
-        setUpMotors();
+        is_initialized = false;
+        setup();
         encoders_calibrated = false;
```

Comment:
Does encoder calibration and checking `encoder_calibrated_` have to be done in `poll`? Can't it be done in `setup`? Since if I understand the logic correctly, we only calibrate at initialization and if a reset was detected.